server.js

```diff
@@ -1,25 +1,19 @@
-app.put('/slambook/:id', async (req, res) => {
+app.delete('/slambook/:id', async (req, res) => {
   const entryId = req.params.id;

```

Comment:
The variable 'entryId' is not defined in the code. Make sure to assign it the value of 'req.params.id'